tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -105,6 +105,9 @@ static const struct device *const devices[] = {
 #ifdef CONFIG_COUNTER_TMR_ESP32
 	DEVS_FOR_DT_COMPAT(espressif_esp32_counter)
 #endif
+#ifdef CONFIG_COUNTER_RTC_ESP32
```

Comment:
Can you move this change from here and move to the next commit? Then re-do commit info so that this commit it is not tied to ESP32 but to the `test` case only.